scripts/run_evolution.py

```
@@ -503,11 +459,11 @@ def evaluate_fitness(  # noqa: PLR0913
     for ep in range(episodes):
         # Seed RNGs for reproducibility when base_seed is provided
         if base_seed is not None:
-            episode_seed = _derive_episode_seed(base_seed, gen, candidate_idx, ep)
+            episode_seed = derive_episode_seed(base_seed, gen, candidate_idx, ep)
             np.random.seed(episode_seed)  # noqa: NPY002
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `NPY002`)

Remove unused `noqa` directive

(RUF100)